src/lib/BasicMain.tsx

```diff
@@ -8,33 +8,28 @@ import {
   Route
 } from 'react-router-dom'
 
-
 import BasicLed from './BasicLed'
 import { Gubu } from 'gubu'
 
-
-function makeCmp(view: any, ctx: any) {
+function makeCmp (view: any, ctx: any) {
   let cmp: any = () => <div>NONE</div>
 
   const content = view.content || {}
 
-  if ('custom' === content.kind) {
+  if (content.kind === 'custom') {
```

Comment:
yoda

Reply:
[`ts-standard`](https://github.com/standard/ts-standard) seems to be the one overwriting yoda conditions. And It doesn't seem it's possible to disable rules on `ts-standard` (according to their docs). 

Instead they recommend using `ESLint` with [`standard/eslint-config-standard-with-typescript`](https://github.com/standard/eslint-config-standard-with-typescript)

Should I switch to using `ESLint` with `standard/eslint-config-standard-with-typescript`? It should give us more flexibility while still leveraging the standard rules.